scripts/generators/es_template.py

```diff
@@ -175,7 +175,7 @@ def entry_for(field):
                 if mf_type == 'keyword':
                     ecs_helpers.dict_copy_existing_keys(mf, mf_entry, ['normalizer', 'ignore_above'])
```

Comment:
why not supporting the ```analyzer``` key here too?

Reply:
Done that. I did not work much with different analyzers in index templates so I was not sure.